### DISCUSSION TITLE
Looks like `/crypto_keyfile.bin` is accessible in all profiles - should it be blacklisted?

### DISCUSSION THREAD

**Discussion by netcarver** [netblue30/firejail#5662] [Q&A] [answered]
Hello, I've just started using firejail and noticed that `/crypto_keyfile.bin` is not being removed from the view of the filesystem that firejailed applications get. For now, I've extended my local profiles to deny access to this file, and this works.

I would have thought this would be a candidate for adding to the blacklisted files in `disable-common.inc` but I really don't know enough about firejail or LUKS configuration yet to comment any more about this.

https://wiki.archlinux.org/title/dm-crypt/Encrypting_an_entire_system

**Comment by ghost** [comment, marked as answer]
> I would have thought this would be a candidate for adding to the blacklisted files in disable-common.inc but I really don't know enough about firejail or LUKS configuration yet to comment any more about this.

Welcome to the Firejail project. While there's always room for learning you seem to do fine! You're assumptions are correct and we should indeed blacklist that file. Can you open a PR to add it in disable-common.inc? At first glance you could create a new entree group `dm-crypt / LUKS` like we have for VeraCrypt for example. Or add it to `top secret`.

**Comment by netcarver** [comment]
@glitsj16 Thank you, [PR opened](https://github.com/netblue30/firejail/pull/5663).
